libs/blocks/table/table.css

```diff
@@ -131,6 +132,21 @@
   flex-wrap: wrap;
 }
 
+.table.header-left .row-heading .col.col-heading {
+  text-align: start;
+  padding: var(--spacing-s);
+}
+
+.table:not(.merch) .row-heading .col-heading .action-area {
```

Comment:
for consistency does this need to be `.col.col-heading` ?

Reply:
@chrischrischris I have made the suggested change.